app/Http/Controllers/Api/AirlineController.php

```diff
@@ -45,4 +52,19 @@ public function destroy(Airline $airline): JsonResponse
             'status' => 'success',
         ]);
     }
+
+    public function destroyCities(Airline $airline): JsonResponse
+    {
+        $airline->cities()->detach(request()->cityIds);
```

Comment:
```suggestion
        $airline->cities()->detach($request->string('cityIds')->toString());
```
Inject the `Request` dependency rather than using the helper.